src/azure-cli-core/azure/cli/core/local_context.py

```diff
@@ -7,47 +7,90 @@
 import stat
 import shutil
 import configparser
+import enum
 
 from knack.log import get_logger
 from knack.config import _ConfigFile
-from knack.util import ensure_dir, CLIError
+from knack.util import ensure_dir
 
-SET = 'SET'   # action for a parameter in local context, SET means its value will be saved to local context
-GET = 'GET'   # action for a parameter in local context, GET means will read value from local context for this parameter
 ALL = 'ALL'   # effective level of local context, ALL means all commands can share this parameter value
+LOCAL_CONTEXT_FILE = '.local_context_{}'
+LOCAL_CONTEXT_CONFIG_SECTION = 'local_context'
```

Comment:
Let's make it clear about the purpose or add a comment. 

```suggestion
LOCAL_CONTEXT_ON_OFF_CONFIG_SECTION = 'local_context'
```

Reply:
Updated